src/ets/scala/io/iohk/ethereum/ets/blockchain/ScenarioSetup.scala

```diff
@@ -43,6 +43,8 @@ object ScenarioSetup {
 
 abstract class ScenarioSetup(_vm: VMImpl, scenario: BlockchainScenario) {
 
+  import BlockchainTestConfig._
```

Comment:
is this needed?

Reply:
It is, as baseBlockchainConfig and withSkippedPoWValidationBlockchainConfig functions use the BlockchainConfig's defined there